File: class2/mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
    int n;
    for(n = get_int("width: ");
    n <1;
    n = get_int("width: "))
    {
    }

    for (int i = 0; i<n; i++)
    {
        printf("?");
    }
    printf("\n");
}

File: class7/homework/mario.py
from cs50 import get_int

height = get_int("Height: ")
if not height in range(1,8):
    print("Enter the height!")
for i in range (height):
    print(" "*(height-i-1), end = "")
    print("#"*(i+1), end = "")
    print(" "*2, end = "")
    print("#"*(i+1))
